WcaOnRails/app/assets/javascripts/currencies.js

```diff
@@ -0,0 +1,77 @@
+window.wca = window.wca || {};
+
+wca.getCurrencyInfo = function(isoCode) {
+  return wca._currenciesInfo[isoCode] || wca._currenciesInfo.USD;
+};
+
+// Create a mask for an amount input in the given currency, using autoNumeric
+// 'action' can be either "init" or "update"
+// 'elemId' is a selector for the targeted input field
+wca.applyCurrencyMask = function(action, elemId, currencyIsoCode) {
+  if (!$(elemId).length)
```

Comment:
This sort of check always freaks me out. Why would we ever pass an invalid id in here?

Unfortunately we don't have JS linting set up, but please always use curly braces with if statements.

I see 3 other constructs similar to this one in this file, please update them accordingly.